app/src/test/java/com/yalta/services/PasswordServiceUnitTest.kt

```diff
@@ -1,28 +1,35 @@
 package com.yalta.services
 
 import com.yalta.CoroutineTestRule
+import com.yalta.repositories.FailedResponse
 import com.yalta.repositories.FakePasswordRepo
+import com.yalta.repositories.PasswordChanged
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.runBlockingTest
 import org.junit.Assert.*
 import org.junit.Rule
 import org.junit.Test
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.mock
 
 @ExperimentalCoroutinesApi
 class PasswordServiceUnitTest {
-    private val repo = FakePasswordRepo()
+    private val test = mock(FakePasswordRepo::class.java)
```

Comment:
In reality you don't need to mock Fake, since it's double work. You can(and should) mock your real class and redefine it's behaviour on the go, but ok, this works